Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1010,15 +1013,24 @@ - (void)didReceiveData:(NSData *)data forTask:(NSURLSessionDataTask *)dataTask
     [self unlock];
     
     [progressiveImage updateProgressiveImageWithData:data expectedNumberOfBytes:[dataTask countOfBytesExpectedToReceive]];
-    if (hasProgressBlocks && [[self class] isiOS8OrGreater]) {
-        UIImage *progressImage = [progressiveImage currentImage];
-        if (progressImage) {
-            [self lock];
-                task = [self.tasks objectForKey:[self cacheKeyForURL:[[dataTask originalRequest] URL] processorKey:nil]];
-                [task callProgressWithQueue:_callbackQueue withImage:progressImage];
-            [self unlock];
+
+    __weak typeof(self) weakSelf = self;
+    [_concurrentOperationQueue pin_addOperationWithQueuePriority:PINRemoteImageManagerPriorityLow block:^{
+        typeof(self) strongSelf = weakSelf;
+        [strongSelf lock];
+            PINRemoteImageDownloadTask *task = [strongSelf.tasks objectForKey:[strongSelf cacheKeyForURL:[[dataTask originalRequest] URL] processorKey:nil]];
+        [strongSelf unlock];
```

Comment:
Is this locked section needed? Two things:

 - This work looks unnecessary if the `hasProgressBlocks && [[strongSelf class] isiOS8OrGreater]` test below fails. 
 - This dictionary lookup looks identical to the one below.